components/event/EventCard.js

```diff
@@ -76,6 +76,14 @@ export default function EventCard({ event, username }) {
               <ReactMarkdown className="text-sm text-gray-500 py-1 flex-wrap">
                 {event.description}
               </ReactMarkdown>
+              <p className="text-sm text-gray-800 py-1 flex-wrap">
+                {event.isVirtual === true && event.isInPerson === false ? (`Remote`) : 
+                event.hasOwnProperty('location') ? (
```

Comment:
is there a specific reason for a conditional statement here as we are either showing it or not
we can simplify it by `event.location && (...code)`

Reply:
Yes, I don't know that. Learnt this now. Thanks, and I will change the code as per requirement. This is helpful.

Reply:
Great collab 👍